File: Lecture2/Constant.c
#include <stdio.h>
#include <string.h>

int main(void) {

    const int x = 100;
    const int y = 200;
    int sum;

    sum = x + y;

    printf("%d\n", sum);


    const char str_x[] = "100";
    const char str_y[] = "200";

    char str_sum[2014];
    strcpy(str_sum, str_x);
    strcat(str_sum, str_y);

    printf("%s\n", str_sum);

    return 0;
}

File: Lecture2/Constant.java
import java.util.Scanner;

public class Constant {
	public static void main(String[] args) {
		final int x = 100;
		final int y = 200;
		int sum;
		
		sum = x + y;
		System.out.println(sum);
		
		final String str_x = "100";
		final String str_y = "200";
		String str_sum;
		
		str_sum = str_x + str_y;
		System.out.println(str_sum);
	}
}